src/main.js

```diff
@@ -0,0 +1,365 @@
+      keyboardRowValues.forEach((keyValue) => {
+        if (keyValue[1] === event.code && !ignoreKeys.includes(event.key)) {
+          event.preventDefault();
+          if (event.key === 'Tab') this.symbol = '  ';
+          else {
+            const [, , ruLow, ruUp, enLow, enUp] = keyValue;
+            switch (this.lang) {
+              case 'en':
+                if (this.isShiftPress || this.isCapsLockPress) this.symbol = enUp;
+                else this.symbol = enLow;
+                break;
+              case 'ru':
+                if (this.isShiftPress || this.isCapsLockPress) this.symbol = ruUp;
+                else this.symbol = ruLow;
+                break;
+              default:
+                break;
+            }
+          }
+        }
+      });
+    });
+
+    // show what we are click to button
+    this.keyboard.querySelectorAll('.row').forEach((row) => {
+      row.querySelectorAll('.key').forEach((key) => {
+        if (event.code === key.children[0].classList[0]) {
+          if (!this.isShiftPress && event.key === 'CapsLock') {
+            this.checkCaps();
+          } else {
+            key.classList.add('active');
+          }
+        }
+      });
+    });
+
+    // print symbol to textarea
+    this.textArea.setRangeText(
+      this.symbol,
+      this.textArea.selectionStart,
+      this.textArea.selectionEnd,
+      'end',
+    );
+  }
+
+  // when key press up use that
+  keyUp(event) {
+    if (this.isCapsLockPress !== true && event.key === 'Shift') {
+      this.isShiftPress = false;
+      this.changeRegister();
+    }
+
+    // show what we are stop clicking to button
+    this.keyboard.querySelectorAll('.row').forEach((row) => {
+      row.querySelectorAll('.key').forEach((key) => {
+        if (event.code === key.children[0].classList[0] && event.key !== 'CapsLock') {
+          key.classList.remove('active');
+        }
+      });
+    });
+  }
+
+  // interactive to mouse click
+  clickPrint(event) {
+    this.textArea.focus();
+
+    this.symbol = '';
+    const targetBtn = event.target.closest('button');
+
+    if (targetBtn) {
+      const targetSpan = targetBtn.querySelector('.on');
+      const targetBtnName = targetSpan.className.split(' ')[0];
+      const specialBtn = targetBtn.classList[1];
+
+      targetBtn.classList.add('active');
+      setTimeout(() => {
+        targetBtn.classList.remove('active');
+      }, 150);
+
+      keyboardKeys.forEach((row) => {
+        row.forEach((keyValue) => {
+          if (keyValue[1] === targetBtnName && (specialBtn === undefined || specialBtn === 'space')) {
+            const [, , ruLow, ruUp, enLow, enUp] = keyValue;
+            switch (this.lang) {
+              case 'en':
+                if (this.isShiftPress || this.isCapsLockPress) this.symbol = enUp;
+                else this.symbol = enLow;
+                break;
+              case 'ru':
+                if (this.isShiftPress || this.isCapsLockPress) this.symbol = ruUp;
```

Comment:
Use brackets for each if statement, it improves readability

Reply:
now exactly got it what that means and fixed =)